lib/spi/addon-contracts.ts

```diff
@@ -1,4 +1,4 @@
-import { Construct } from '@aws-cdk/core';
+import { Construct } from "constructs";
```

Comment:
quotes

Reply:
need a linting rule to catch all of this @shapirov103

Reply:
agree. will defer, not the highest priority task, but will throw an issue to the backlog.